src/Microsoft.Diagnostics.Monitoring.WebApi/EncodingCache.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    internal static class EncodingCache
+    {
+        public static readonly Encoding UTF8NoBOM =
+            new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
```

Comment:
Do you know the practical impact of the second parameter? For example, can someone emit a log statement that contains invalid bytes and we end up crashing as a result when we attempt to encode the payload?